pipcompilemulti/environment.py

```diff
@@ -63,11 +63,23 @@ def create_lockfile(self):
             if sink_out_path and sink_out_path != self.outfile:
                 original_in_file = self._read_infile()
                 self._inject_sink()
+
+            verbose = True
             process = subprocess.Popen(
                 self.pin_command,
                 stdout=subprocess.PIPE,
```

Comment:
I think the stdout is captured only for logging it later, so we could just set: `stdout=None` and then skip logging stdout in verbose mode

Reply:
Setting the `stdout=None` here gives:

```
Traceback (most recent call last):
  File "/Users/john/.virtualenvs/pip-compile-multi-demo/lib/python3.8/site-packages/pipcompilemulti/cli_v1.py", line 26, in cli
    recompile()
  File "/Users/john/.virtualenvs/pip-compile-multi-demo/lib/python3.8/site-packages/pipcompilemulti/actions.py", line 30, in recompile
    compile_topologically(env_confs, deduplicator)
  File "/Users/john/.virtualenvs/pip-compile-multi-demo/lib/python3.8/site-packages/pipcompilemulti/actions.py", line 39, in compile_topologically
    if env.maybe_create_lockfile():
  File "/Users/john/.virtualenvs/pip-compile-multi-demo/lib/python3.8/site-packages/pipcompilemulti/environment.py", line 50, in maybe_create_lockfile
    self.create_lockfile()
  File "/Users/john/.virtualenvs/pip-compile-multi-demo/lib/python3.8/site-packages/pipcompilemulti/environment.py", line 74, in create_lockfile
    output = process.stdout.readline()
AttributeError: 'NoneType' object has no attribute 'readline'
```

Reply:
Yeah you won't need to use process.stdout anymore as it should be sent to the terminal already

Reply:
Oh my that makes it simpler